app/renderer/components/navigation/publisherToggle.js

```diff
@@ -66,7 +66,7 @@ class PublisherToggle extends React.Component {
       return
     }
     let updateTimeout = setTimeout(() => {
-      appActions.onPublisherToggleUpdate()
```

Comment:
let's remove the whole flow `appActions.onPublisherToggleUpdate()` as it's not used anywhere else